Failed to apply upload policy

This error is seen while uploading file to s3
```

ERROR [Thread-6] (Consumer.java:172) - Thread failed
java.lang.RuntimeException: Failed to apply upload policy
	at com.pinterest.secor.consumer.Consumer.checkUploadPolicy(Consumer.java:182)
	at com.pinterest.secor.consumer.Consumer.run(Consumer.java:163)
Caused by: com.amazonaws.AmazonClientException: Unable to complete transfer: com.amazonaws.util.IOUtils.release(Ljava/io/Closeable;Lcom/amazonaws/thirdparty/apache/logging/Log;)V
	at com.amazonaws.services.s3.transfer.internal.AbstractTransfer.unwrapExecutionException(AbstractTransfer.java:286)
	at com.amazonaws.services.s3.transfer.internal.AbstractTransfer.rethrowExecutionException(AbstractTransfer.java:265)
	at com.amazonaws.services.s3.transfer.internal.UploadImpl.waitForUploadResult(UploadImpl.java:66)
	at com.pinterest.secor.uploader.S3UploadHandle.get(S3UploadHandle.java:38)
	at com.pinterest.secor.uploader.S3UploadHandle.get(S3UploadHandle.java:30)
	at com.pinterest.secor.uploader.Uploader.uploadFiles(Uploader.java:133)
	at com.pinterest.secor.uploader.Uploader.checkTopicPartition(Uploader.java:281)
	at com.pinterest.secor.uploader.Uploader.applyPolicy(Uploader.java:320)
	at com.pinterest.secor.consumer.Consumer.checkUploadPolicy(Consumer.java:180)
	... 1 more
Caused by: java.lang.NoSuchMethodError: com.amazonaws.util.IOUtils.release(Ljava/io/Closeable;Lcom/amazonaws/thirdparty/apache/logging/Log;)V
	at com.amazonaws.services.s3.model.S3DataSource$Utils.cleanupDataSource(S3DataSource.java:48)
	at com.amazonaws.services.s3.AmazonS3Client.putObject(AmazonS3Client.java:1726)
	at com.amazonaws.services.s3.transfer.internal.UploadCallable.uploadInOneChunk(UploadCallable.java:133)
	at com.amazonaws.services.s3.transfer.internal.UploadCallable.call(UploadCallable.java:125)
	at com.amazonaws.services.s3.transfer.internal.UploadMonitor.call(UploadMonitor.java:143)
	at com.amazonaws.services.s3.transfer.internal.UploadMonitor.call(UploadMonitor.java:48)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

Conf: `secor.upload.manager.class=S3UploadManager`, I could see this is caused by hadoop-aws dep which is bundling the same.

--- a/pom.xml
+++ b/pom.xml
@@ -241,6 +241,10 @@
                     <groupId>com.amazonaws</groupId>
                     <artifactId>aws-java-sdk</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.amazonaws</groupId>
+                    <artifactId>aws-java-sdk-bundle</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>

